modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/util/KBDropdownItemsProvider.java

```diff
@@ -1019,6 +1053,18 @@ private boolean _hasImportPermission(KBFolder kbFolder)
 		return false;
 	}
 
+	private boolean _hasKBChildArticles(KBArticle kbArticle) {
+		int childKBArticlesCount = KBArticleServiceUtil.getKBArticlesCount(
+			_themeDisplay.getScopeGroupId(), kbArticle.getResourcePrimKey(),
+			WorkflowConstants.STATUS_ANY);
+
+		if (childKBArticlesCount > 0) {
+			return true;
+		}
+
+		return false;
+	}
+
```

Comment:
I would rename this method to `_hasChildArticles` and the variable to `childArticlesCount`.